src/status_im2/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -35,7 +35,13 @@
 (defn view
   []
   (let [{id :community-id}          (rf/sub [:get-screen-params])
-        {:keys [name color images]} (rf/sub [:communities/community id])]
+        {:keys [name color images]} (rf/sub [:communities/community id])
+        {:keys [accounts]}          (rf/sub [:wallet])
+        account-list                (->> accounts
```

Comment:
Do we really need `account-list` to be an instance of PersistentVector? I saw that `label-props` is used with the Clojure's `for` macro, so it can be any sequence.